demo/locales/en.json

```diff
@@ -2,6 +2,7 @@
   "DEMO_ADAPTATION_RESTRICTIONS_SECTION_HEADER": "Adaptation Restrictions",
   "DEMO_ADAPTATION_SECTION_HEADER": "Adaptation",
   "DEMO_ADS": "Ads",
+  "DEMO_AD_TAG_URL": "Ad Tag URL",
```

Comment:
These lists are supposed to be alphabetized now. So this should come after `DEMO_AD_SEARCH`, but before `DEMO_ALWAYS_STREAM_TEXT`.

Reply:
Fixed!